cmd/subcommands/transfer.go

```diff
@@ -255,6 +256,11 @@ func handlerForBulkTransactions(txLog *transactionLog, index int) error {
 	}
 	trueNonce = txnFlags.TrueNonce
 
+	// Wait for 10 sec for intra-shard transfer
+	if fromShardID == toShardID && timeout != defaultTimeout {
```

Comment:
We set the timeout to 0 to send a transaction without checking, adding this check would break that behavior. Maybe add a check for 0 timeout?

Also, this only applies to bulk transactions, and should also apply to regular transactions. Maybe move this to `RunE` of the transfer command?